documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -184,6 +184,20 @@ endif::[]
 |
 | Set this parameter to an available IP address on the `baremetal` network when the `provisioningNetwork` configuration setting is set to `Disabled`.
 
+ifeval::[{release} > 4.4]
+| `httpProxy`
+|
+| Set this parameter to the appropriate http_proxy used within your environment.
```

Comment:
should we remove the underscore "_" between http and proxy? meaning using it as a sentence instead of 'var name' ?